bin/drop_sample_annot.py

```diff
@@ -8,36 +8,40 @@
 import os
 import pandas as pd
 
+SAMPLE_ANNOTATION_COLUMNS = [
+    "RNA_ID",
+    "RNA_BAM_FILE",
+    "DNA_VCF_FILE",
+    "DNA_ID",
+    "DROP_GROUP",
+    "PAIRED_END",
+    "COUNT_MODE",
+    "COUNT_OVERLAPS",
+    "SPLICE_COUNTS_DIR",
+    "STRAND",
+    "HPO_TERMS",
+    "GENE_COUNTS_FILE",
+    "GENE_ANNOTATION",
+    "GENOME",
+]
+
 
 class SampleAnnotation:
     """SampleAnnotation class"""
```

Comment:
Add a more descriptive docstring or remove it if it doesn't add more information

Reply:
Done